### PR TITLE
Command line arguments for FrameRipper

### PR DESCRIPTION
**Pull request by tlacasse**
allow the frameripper to be "more" generic, by having videos path (and optionally frame ripping count) passed as command line arguments

use like:
> cd src
> python FrameRipper.py VIDEOSPATH [--framecount FRAMECOUNT] 

Arguments:
- `videospath` - path to videos to rip and crop frames
- `framecount` *(optional)* - number of frames to rip for each video

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,8 @@
 *.png
 *.pyc
 *.mov
+src/arc/*
+src/good_stuff/*
```
Comment (NickMuise) [inline .gitignore:7]: what

**Review by NickMuise — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (tlacasse) [reply .gitignore:7]: do you want to `fight`

Comment (NickMuise) [reply .gitignore:7]: no I want to `fuck`

Comment (tlacasse) [reply .gitignore:7]: `yeehaw` it is then

Comment (NickMuise) [reply .gitignore:7]: It's real `yeehaw` hours here

**Pull request merged**